explorer/interpreter/interpreter.cpp

```diff
@@ -2706,21 +2706,28 @@ auto Interpreter::StepCleanUp() -> ErrorOr<Success> {
 
 // State transition.
 auto Interpreter::Step() -> ErrorOr<Success> {
+  Action& act = todo_.CurrentAction();
+
+  auto fatal_error_builder = [&] {
```

Comment:
All the errors are "fatal", but not typically labeled as such -- this made me confused briefly versus CARBON_FATAL. Maybe just error_builder or choose_error_builder? It's too bad this needs to be a lambda instead of a variable...

Reply:
Done.